src/Uno.UI/UI/Xaml/Extensions/FontHelper.Lookup.Android.cs

```diff
@@ -55,19 +57,19 @@ public override bool Equals(object other)
 
 			private readonly HashtableEx _entries = new HashtableEx();
 
-			internal bool TryGetValue(Entry key, out Typeface result)
+			internal bool TryGetValue(Entry key, out Typeface? result)
```

Comment:
How about add `[NotNullWhen(true)]` attribute?

See: [C# Reserved attributes: Nullable static analysis | Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/attributes/nullable-analysis#conditional-post-conditions-notnullwhen-maybenullwhen-and-notnullifnotnull )

[NotNullWhenAttribute Class (System.Diagnostics.CodeAnalysis) | Microsoft Docs](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.codeanalysis.notnullwhenattribute?view=net-5.0 )

Reply:
It really can be null if there's no typeface available for the key.